zebra/zebra_routemap.c

```diff
@@ -388,7 +388,7 @@ static int ip_nht_rm_del(struct zebra_vrf *zvrf, const char *rmap, int rtype,
 					zvrf->vrf->vrf_id, rtype);
 			NHT_RM_MAP(zvrf, afi, rtype) = NULL;
 
-			zebra_evaluate_rnh(zvrf, AFI_IP, 1, NULL, SAFI_UNICAST);
+			zebra_evaluate_rnh(zvrf, afi, 1, NULL, SAFI_UNICAST);
```

Comment:
please fix line 1706 as well

Reply:
sorry, I don't know how to add it to this pull request, I create a new one.
https://github.com/FRRouting/frr/pull/14025/files